content/_index.html

```diff
@@ -0,0 +1,495 @@
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <div class="top-slant-container">
+      <div class="top-slant-down has-background-rapids-purple-primary"></div>
+    </div>
+
+    <!-- Quick Start -->
+    <div id="quick-start"></div>
+    <section class="section has-background-rapids-purple-primary has-text-white padding-bottom-slant">
+      <div class="container">
+        <h1 class="mb-5 has-text-white">Quick Start</h1>
+        <div class="columns">
+          <div class="column is-half">
+            <h2 class="has-text-white"><i class="fa-light fa-download"></i> Quick Local Install </h2>
+            <p>RAPIDS offers several installation methods, the quickest is shown below. For more detailed guides go to
+              our <a href="https://docs.rapids.ai/install">advanced install page <i
+                  class="fa-solid fa-arrow-up-right"></i></a> </p>
+
+            <h3 class="mt-5 has-text-white">Requirements</h3>
+            <p><strong>A.</strong> NVIDIA Pascal™ or better GPU with a <a href="https://developer.nvidia.com/cuda-gpus"
+                taget="_blank">compute capability 6.0 and above <i class="fa-solid fa-arrow-up-right"></i></a></p>
+            <p><strong>B.</strong> Ubuntu 20.04 or 22.04, CentOS 7, Rocky Linux 8, or WSL2 on Windows 11</p>
+            <p><strong>C.</strong> Recent <a href="https://docs.nvidia.com/deploy/cuda-compatibility/index.html"
+                target="_blank"> CUDA version <em>and</em> NVIDIA driver pairs</a>. Check yours with:
+              <code>nvidia-smi</code>
+
+            <h3 class="mt-6 has-text-white">Install with Conda</h3>
+            <p><strong>1.</strong> Download and run the conda install script: </p>
+            <pre class="highlight use-white-copy"><code>wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh</code></pre>
+            <br>
+            <p><strong>2.</strong> Then quick install RAPIDS with:</p>
+            <pre
+              class="highlight use-white-copy"><code>conda create -n rapids-23.02 -c rapidsai -c conda-forge -c nvidia  \ rapids=23.02 python=3.10 cudatoolkit=11.8</code></pre>
+
+            <h3 class="mt-6 has-text-white">Install with pip</h3>
+            <p><strong>1.</strong> Install via pip channels:</p>
+            <pre class="highlight use-white-copy"><code>pip install cudf-cu11 dask-cudf-cu11 --extra-index-url=https://pypi.nvidia.com 
+pip install cuml-cu11 --extra-index-url=https://pypi.nvidia.com
+pip install cugraph-cu11 --extra-index-url=https://pypi.nvidia.com</code></pre>
+
+            <h3 class="mt-6 has-text-white">Install with Docker</h3>
+            <p>Use the <a href="https://docs.rapids.ai/install#docker">install page selector <i
+                  class="fa-solid fa-arrow-up-right"></i> </a></p>
+
```

Comment:
This links to https://docs.rapids.ai/install#docker , which is correct but feels a bit confusing as a UX.

When I clicked it, I expected to see the docker selector widget rather than direction for how to install NVIDIA Docker and then WSL2 instructions. Farther down on the page I saw the selector, but it was confusing.

I think installing via Docker is standard enough that it makes sense to directly go to the release selector. What do you think?

Reply:
Why note both!